src/utils.js

```diff
@@ -65,6 +65,24 @@ export function debounce(func, wait) {
     }
 }
 
+const handleError = (el, expression, error) => {
+    console.error(`Alpine: error in expression "${expression}" in component:`, el, `due to "${error}"`);
+    if (! isTesting()) {
+        throw error;
+    }
```

Comment:
Avoid throwing during tests eg. when testing event handlers if we throw it's basically untestable